src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/proposal/quickfix/RemoveMultipleAnnotations.java

```diff
@@ -55,17 +56,21 @@ public List<? extends CodeAction> getCodeActions(JavaCodeActionContext context,
 
             List<List<String>> annotationsListsToRemove = getMultipleRemoveAnnotations(parentType.getProject(), annotations);
             for (List<String> annotationList : annotationsListsToRemove) {
+
                 // For each list we will create one code action in its own context
```

Comment:
Should delete or change this comment since this code no longer copies the context.